src/Microsoft.DotNet.XHarness.CLI/Microsoft.DotNet.XHarness.CLI.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
I am thinking about the other projects.. There are some dependent nuget packages such as `Microsoft.DotNet.XHarness.Common`. Maybe other projects could be updated too?

Reply:
That would not run in the `net6.0` process no ? (unless you mean we should add targets, not replace it)
Is `net6.0` xharness still in use ?

Reply:
net6.0 is still lts and you can target all of them from the sdk